src/xunit.netcore.extensions/Discoverers/ActiveIssueDiscoverer.cs

```diff
@@ -28,17 +28,46 @@ public class ActiveIssueDiscoverer : ITraitDiscoverer
             Debug.Assert(ctorArgs.Count() >= 2);
 
             string issue = ctorArgs.First().ToString();
-            TestPlatforms platforms = (TestPlatforms)ctorArgs.Last();
+            TestPlatforms platforms = TestPlatforms.Any;
+            TargetFrameworkMonikers frameworks = TargetFrameworkMonikers.All;
+            
+            if (ctorArgs.Count() > 2)
+            {
+                platforms = (TestPlatforms)ctorArgs.ElementAt(1);
```

Comment:
Why make this assumption? Why not just loop over the arguments and they try casting each argument to both types and do the work based on which type it is?